README.md

```diff
@@ -1,142 +1,17 @@
 Ledge-oe-manifest
-=================
-(derivated from https://github.com/96boards/oe-rpb-manifest)
 
-LEGDE OE Repo manifest repository
+This is the main manifest file to build the LEDGE Reference Platform from the source. For detailed build environment
+and build instruction please refer to ledge-dev-howto.pdf (generated from https://github.com/Linaro/ledge-doc
+sources).
 
-These are the setup scripts for the LEDGE OE buildsystem. If you want to (re)build packages or images for OE LEGDE, this is the thing to use.
-The OE LEDGE buildsystem is using various components from the Yocto Project, most importantly the Openembedded buildsystem, the bitbake task executor and various application and BSP layers.
+For advanced users quick steps to build are:
+1. repo init -u https://github.com/Linaro/ledge-oe-manifest.git -b master
+2. MACHINE=ledge-multi-armv7 DISTRO=rpb source ./setup-environment build-rpb-mc
+3. Follow printed instructions to build ledge-iot and ledge-gateway images.
+Note: supported MACHINEs are: ledge-multi-armv7, ledge-multi-armv8, ledge-qemux86-64 
 
```

Comment:
what about the ST board?

Reply:
@apalos  ST board is also under ledge-multi-armv7